perma_web/frontend/components/CaptureError.vue

```diff
@@ -58,7 +62,7 @@ defineExpose({
                 </span></p>
             <p v-if="showGeneric" class="message">We’re unable to create your Perma Link.</p>
             <template v-if="showUploadLink">
-                <template v-if="showDevPlayground">
+                <template v-if="false && showDevPlayground">
```

Comment:
On purpose but I think the wrong decision -- my thinking was since the upload form isn't done I should disable it, but if you're QAing with developer-playground = true you'll want to see the new one, done or not. Pushed a fix.